frontend/src/app/modules/common/autocomplete/create-autocompleter.component.ts

```diff
@@ -27,27 +27,34 @@
 // ++
 
 import {AfterViewInit, ChangeDetectorRef, Component, EventEmitter, Input, Output, ViewChild} from '@angular/core';
+
+export interface CreateAutocompleterValueOption {
+  name:string;
+  $href:string|null;
+}
 import {DynamicBootstrapper} from "core-app/globals/dynamic-bootstrapper";
 import {NgSelectComponent} from "@ng-select/ng-select";
 import {I18nService} from "core-app/modules/common/i18n/i18n.service";
 import {CurrentProjectService} from "core-components/projects/current-project.service";
 import {PathHelperService} from "core-app/modules/common/path-helper/path-helper.service";
 import {HalResource} from "core-app/modules/hal/resources/hal-resource";
 import {AddTagFn} from "@ng-select/ng-select/lib/ng-select.component";
+import { Subject } from 'rxjs';
 
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/opf/openproject/blob/c707e7bf78dbad7cac984139a18fb9d109e530df/frontend/src/app/modules/common/autocomplete/create-autocompleter.component.ts#L43"></span>Please use <code>ChangeDetectionStrategy.OnPush</code> for this component</td>
    </tr>
  </tbody>
</table>